tests/provider/dwd/observation/test_fileindex.py

```diff
@@ -39,7 +40,17 @@ def test_file_index_creation():
         "climate/daily/kl/recent/tageswerte_KL_01048_akt.zip"
     ]
 
-    with pytest.raises(requests.exceptions.HTTPError):
+
+@pytest.mark.remote
+def test_file_index_creation_failure():
+
+    with pytest.raises(
+        (
+            requests.exceptions.HTTPError,
+            aiohttp.client_exceptions.ClientResponseError,
```

Comment:
I think we should only allow for FileNotFoundError here, because if the server cant be reached that would be interesting for us.

Reply:
I am okay with that, 6d658c983 addresses your suggestion.